index.html

```diff
@@ -640,6 +641,38 @@ <h2>Typography</h2>
 				<h2>Spinner</h2>
 				<ts-spinner data-visible data-size="medium" data-color="mono"> </ts-spinner>
 			</article>
+
+			<article>
+				<h2>Search</h2>
+				<div style="display: flex;">
+					<div style="width: 300px; margin: 5px;">
```

Comment:
It's fine but you can also use the index.css file for styling this page

Reply:
Oh, I didn't see this file. I moved the styles to index.css.